oaknational/Oak-Web-Application
# Feat/pupil 1005/create results view

## Description

Music year: 1983

### Job Story

**When I complete a lesson**,
**I want to print my results**
**So that I can share my lesson attempt with the teacher**.

### Acceptance criteria

- [x]  attemptData is passed as a prop
- [x]  all relevant data visible on page (unstyled)

## Issue(s)

Fixes #

## How to test

1. Go tohttps://deploy-preview-2778--owa-storybook.netlify.thenational.academy/?path=/story/components-pupilviews-pupilresults--default  (not sure why storybook isnt rendering the mathjax - it is for me locally)
2. Click on results
3. You should see a NONE STYLES printable results page with all the printable results

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
